src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -720,7 +721,7 @@ class DocBaseViewer extends BaseViewer {
      * @return {void}
      */
     resize() {
-        if (!this.pdfViewer || !this.pdfViewer.pageViewsReady) {
+        if (!this.pdfViewer || !this.somePageRendered) {
```

Comment:
`this.pdfViewer.pageViewsReady` was updating way after a page was rendered. This new check waits for a rendered event from PDF.js so we know the pdfViewer is inter-actable.